### PR TITLE
feat(Flow): allow nodes to accept inputs from all other nodes

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

File: packages/lab/src/components/Flow/stories/Usage.stories.mdx [hunk 1]
```diff
@@ -223,7 +223,8 @@ type HvFlowNodeOutput = {
 You'll notice that input nodes have an `accepts` property and the output nodes have a `provides` property. These are used to limit
 how nodes connect to other nodes and as a descriptor of what a node provides or accepts. Thus, for example, if a node
 provides `"config"`, only nodes that accept this string will allow a connection from that node. While inputs can accept an array of strings,
-an output only provides one.
+an output only provides one. If, instead, you want a node input to accept outputs from any other node without restrictions,
+you only need to set the `accepts` property to an empty array or, alternativel, not setting the `accepts` at all.
```
Comment (MEsteves22) [inline packages/lab/src/components/Flow/stories/Usage.stories.mdx:227]: Typo: "alternatively"

**Review by MEsteves22 — APPROVED** [review]
Looks good 🙏 We just have a small typo:

File: packages/lab/src/components/Flow/stories/Usage.stories.mdx (hunk 1 above)
Comment (plagoa) [reply packages/lab/src/components/Flow/stories/Usage.stories.mdx]: typo fixed

File: packages/lab/src/components/Flow/stories/Usage.stories.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Meta, Canvas, Story } from "@storybook/addon-docs";
 
-<Meta title="Guides/Flow" />
+<Meta title="Lab/Flow" />
```
Comment (zettca) [inline packages/lab/src/components/Flow/stories/Usage.stories.mdx:3]: This leads to a collision in the docs

```suggestion
<Meta title="Lab/Flow/Usage" />
```

**Review by zettca — APPROVED** [review]

**Pull request merged**